src/main/java/bisq/core/dao/vote/issuance/consensus/IssuanceConsensus.java

```diff
@@ -15,68 +15,89 @@
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 import lombok.extern.slf4j.Slf4j;
 
+import javax.annotation.Nullable;
+
 @Slf4j
 public class IssuanceConsensus {
 
     //TODO set higher value
-    private static final int QUORUM = 1000; // 10 BSQ
-
-    public static SecretKey getSecretKey(byte[] opReturnData) {
-        byte[] secretKeyAsBytes = Arrays.copyOfRange(opReturnData, 22, 54);
-        return Encryption.getSecretKeyFromBytes(secretKeyAsBytes);
-    }
+    private static final int QUORUM = 100; // 1 BSQ
 
     // Hash of the list of Blind votes is 20 bytes after version and type bytes
     public static byte[] getBlindVoteListHash(byte[] opReturnData) {
         return Arrays.copyOfRange(opReturnData, 2, 22);
     }
 
-    //TODO share with vote consensus
-    public static byte[] getHashOfEncryptedProposalList(byte[] encryptedProposalList) {
-        return Hash.getSha256Ripemd160hash(encryptedProposalList);
+    public static byte[] decryptProposalList(byte[] encryptedProposalList, SecretKey secretKey) throws CryptoException {
+        return Encryption.decrypt(encryptedProposalList, secretKey);
+    }
+
+    @Nullable
+    public static byte[] getMajorityHash(List<IssuanceService.HashWithTxIdList> list) {
+        list.sort(Comparator.comparingInt(o -> o.getTxIds().size()));
+        return !list.isEmpty() ? list.get(0).getHashOfProposalList() : null;
     }
 
-    public static void applyVoteResult(Map<Proposal, Integer> stakeByProposalMap,
+    // Key is stored after version and type bytes and list of Blind votes. It has 32 bytes
+    public static SecretKey getSecretKey(byte[] opReturnData) {
+        byte[] secretKeyAsBytes = Arrays.copyOfRange(opReturnData, 22, 54);
+        return Encryption.getSecretKeyFromBytes(secretKeyAsBytes);
+    }
+
+    public static void applyVoteResult(Map<ProposalPayload, Long> stakeByProposalMap,
                                        ReadableBsqBlockChain readableBsqBlockChain,
                                        WritableBsqBlockChain writableBsqBlockChain) {
         Map<String, TxOutput> txOutputsByTxIdMap = new HashMap<>();
+
         final Set<TxOutput> compReqIssuanceTxOutputs = readableBsqBlockChain.getCompReqIssuanceTxOutputs();
         compReqIssuanceTxOutputs.stream()
-                .filter(txOutput -> !txOutput.isVerified()) // our candidate is not yet verified and not set
+                .filter(txOutput -> !txOutput.isVerified()) // our candidate is not yet verified
                 /*.filter(txOutput -> txOutput.isUnspent())*/ // TODO set unspent and keep track of it in parser
                 .forEach(txOutput -> txOutputsByTxIdMap.put(txOutput.getTxId(), txOutput));
 
-        stakeByProposalMap.forEach((proposal, value) -> {
-            int stakeResult = value;
+        stakeByProposalMap.forEach((proposalPayload, value) -> {
+            long stakeResult = value;
             if (stakeResult >= QUORUM) {
```

Comment:
Shouldn't `QUORUM` be compared to the total stake that voted for a proposal (ignoring stake that chose not to vote for a proposal) and then there is issuance if `total stake for a proposal > total stake against`. Currently it seems the sum of votes is compared to `QUORUM` which doesn't really make sense to me.

Reply:
Quorum should be the total number of votes that voted on a proposal, the sum of accepted and rejected, whereas you're now comparing it to the difference between accepted and rejected votes. I would keep the accepted and rejected sums separated and then do something like
```
boolean isAccepted(accepted, rejected) {
  return accepted + rejected >= QUORUM && accepted > rejected;
}
```

Reply:
Yes for the concept of a quorum you are right, we should count up all votes and check if they exceed the quorum independent if the result. Though I am not sure if we should also require a min. threshold of accepted stake. So additionally we would require that total result (accepted-rejected) is larger then a minThreshold. Not 100% sure if that is wanted feature but I think it might be good to let very close results trigger a change (specially or param changes). E.g. 2 big stakeholders vote with 10 000 BSQ pro and contra a proposal. Then a third stakeholder with 1 BSQ could make the decision if accepted or not. I think that's not good.
Also atm we have the stake plain text in the Blind Vote. Not sure if that could be abused and if we should put it into the encrypted data.

Reply:
But if we use the threshold it covers the quorum as well as long it is larger than the quorum.

Reply:
It is also the question if an ignored or not reaching quorum proposal has any different treatment than a rejected proposal. Atm I don't see any use case where we treat it different (e.g. we might not allow to re-publish rejected but we might allow to republish ignored proposals -  but that is not planned yet and I think not needed to specify). So any proposal not reaching enough support will fail, independent if lack of support was due lack of interest (ignore) or because of too much rejection.

Reply:
You're right that there might be a requirement for supermajority for some votes. To handle different majority requirement perhaps it's better to do something like first check for `accepted + rejected >= QUORUM` and then check the percentage vote `accepted / (accepted + rejected) > threshold` as the requirement for an accepted vote.

The threshold doesn't necessarily have to be the same for all types of votes. Compensation requests probably could do with `threshold=0.5` whereas changing fundamental parameters might need more.

Reply:
Yes, true. I will change it that way.